support/cas-server-support-oidc/src/test/java/org/apereo/cas/oidc/token/OidcJwtAccessTokenEncoderTests.java

```diff
@@ -66,6 +66,17 @@ void verifyExtractionAsParameterForService() throws Throwable {
         assertEquals(accessToken.getId(), decoded);
     }
 
+    @Test
+    void verifyNonEncodedToken() throws Throwable {
+        val accessToken = getAccessToken();
+        val registeredService = getRegisteredServiceForJwtAccessTokenWithKeys(accessToken);
+        val encoder = getAccessTokenEncoder(accessToken, registeredService);
+
+        val nonEncodedAccessToken = "nodotintoken";
+        val decode = encoder.decode(nonEncodedAccessToken);
+        assertEquals(nonEncodedAccessToken, decode);
```

Comment:
If one does not already exist, please turn this into a parameterized test where you test equality for:

1. nodotintoken
2. Blank
3. Null